### PR TITLE
Configure dependabot

### LINKED ISSUE #619: Configure dependabot
- [ ] Create own config file https://dependabot.com/docs/config-file/
- [ ] Run updates for trikRuntime weekly and require review
- [ ] Update other submodules automatically on CI success

### PR DESCRIPTION
**Pull request by khodand**
Closes #619

### CODE REVIEW

**Comment by khodand** [comment]
Also, some submodules can be automatically merged, but I have some troubles with synchronizing this action with CI's successfulness.

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+update_configs:
+  - package_manager: "submodules"
+    directory: "plugins/robots/thirdparty/trikRuntime/trikRuntime"
+    update_schedule: "weekly"
+    default_reviewers:
+      - "iakov"
+
+  - package_manager: "submodules"
+    directory: "qrtest/thirdparty/googletest/googletest"
+    update_schedule: "weekly"
```
Comment (iakov) [inline .dependabot/config.yml]:
```suggestion
    update_schedule: "monthly"
```

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  - package_manager: "submodules"
+    directory: "qrgui/thirdparty/qt-solutions"
+    update_schedule: "weekly"
```
Comment (iakov) [inline .dependabot/config.yml]:
```suggestion
    update_schedule: "monthly"
```

File: .dependabot/config.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    directory: "thirdparty/gamepad/gamepad"
+    update_schedule: "weekly"
+
+  - package_manager: "submodules"
+    directory: "installer/nxt-tools"
+    update_schedule: "weekly"
+
+  - package_manager: "submodules"
+    directory: "plugins/robots/thirdparty/hidapi"
+    update_schedule: "weekly"
```
Comment (iakov) [inline .dependabot/config.yml]:
```suggestion
    update_schedule: "monthly"
```

File: .dependabot/config.yml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  - package_manager: "submodules"
+    directory: "thirdparty/quazip/quazip"
+    update_schedule: "weekly"
```
Comment (iakov) [inline .dependabot/config.yml]:
```suggestion
    update_schedule: "monthly"
```

File: .dependabot/config.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+  - package_manager: "submodules"
+    directory: "plugins/robots/thirdparty/Box2D/Box2D"
+    update_schedule: "weekly"
```
Comment (iakov) [inline .dependabot/config.yml]:
```suggestion
    update_schedule: "monthly"
```

**Pull request merged**